codecov.yml

```diff
@@ -3,6 +3,8 @@ coverage:
     project:
       default:
         threshold: 1%
+    patch: off
```

Comment:
Does this mean we will always test code coverage against the entire project? If so, will it cause longer test time for codecov?

Reply:
No, it means that it will not measure what percentage of lines of the patch is modifying are covered, see https://docs.codecov.io/docs/commit-status#patch-status